Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+
+The **Scan** (qr code) button on the main menu provides the most automated methods for importing seeds, using your camera, the cut-and-paste clipboard, the file system, or your photo roll. 
+
+To scan a QR code, you can either point your camera at the QR of a seed, import a QR of a seed from your Photos, or choose an image of a QR from your Files. 
+
+To scan text defining a seed, just copy it into your Clipboard and click Paste. 
+
+Note that for these methodologies, **Seed Tool** expects the QR code or the clipboard to contain a [Uniform Resource](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md), a standardized way to encode data in an efficient and self-identifying way. This will usually mean a [`ur-crypto-seed`](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-006-urtypes.md#cryptographic-seed-crypto-seed).
+
+Besides scanning seeds, you can also use this view to scan SSKR shares (See "Importing SSKR Shares") or respond to a `ur:crypto-request` (see "Answering Requests"), as described below.
+
+### Importing a Seed via Cut and Paste
+
+The **add** ("+") button below the main menu gives a number of options for creating seeds in **Seed Tool**, but also lets you input text as hex bytes, `ur:crypto-seed`, Bytewords, or BIP39 Mnemonic Words. In each case, you just type or paste the words, and then click "Done".
+
+The following show examples of the data you might input for each data type:
+
+* **Hex Bytes:** 59F2293A5BCE7D4DE59E71B4207AC5D2
+* **`ur:crypto-seed`:** `ur:crypto-seed/oyadgdhkwzdtfthptokigtvwnnjsqzcxknsktdhpyljeda`
+* **Byte Words:** hawk whiz diet fact help taco kiwi gift view noon jugs quiz crux kiln silk tied omit keno lung jade
+* **BIP 39 Mnemonic Words:** fly mule excess resource treat plunge nose soda reflect adult ramp planet
+
+Again, you can also add add SSKR shares, as described below.
+
+### Importing SSKR Shares
+
+<a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-import.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-import.jpeg" align="right" width=250></a>
+
+SSKR stands for Sharded Secret Key Reconstruction, a Blockchain Commons [specification](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-011-sskr.md). It allows you to shard a secret (such as a seed) into a number of shares, and then reconstruct the secret from some number (threshold) of those seeds that's typically fewer than all of them. For example, you might shard a seed into three shares with a threshold of two. Typically, the shares are given out to friends and family, placed in distant safety deposit boxes, or otherwise separated. A threshold of the shares can then be collected and used to reconstruct the seed if the original is lost. 
```

Comment:
More typically is that shards are put in different locations, recoverable solely by the user. Shards can also be sused for social recovery as well, but often these are a different sskr set. 

Later we should write up a separate document on some different sskr strategy, and their strengths and weaknesses, in particular the 4 of 9 social recovery scheme that we wrote up for Bitmark.

Reply:
A. I added the difference between social and self-sovereign key recovery.
B. I listed the (new) base scenarios in the guide.
C. I made a note that we're working on a "Scenario Guide for SSKR Shares" and added that to my TODO list.